solarized-definitions.el

```diff
@@ -215,6 +215,8 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
           ;; FIXME: don’t hardcode the SRGB color names
           (fmt-curl-red    `(                        :underline (:color "#dc322f" :style wave)))
           (fmt-curl-yellow `(                        :underline (:color "#b58900" :style wave)))
+          (fmt-curl-magenta `(                      :underline (:color "#d33682" :style wave)))
+          (fmt-curl-cyan `(                      :underline (:color "#2aa198" :style wave)))
```

Comment:
This looks good. Just some stylistic requests.

Could you line up `:underline` with the other ones?